third_party/move/move-prover/move-docgen/src/docgen.rs

```diff
@@ -2148,6 +2152,16 @@ impl<'env> Docgen<'env> {
         }
     }
 
+    /// Determines modules for which we do not create auto links.
```

Comment:
this seems not relevant to this PR?

Reply:
If I do not do this, I cannot use `result` as a module name, see comment below. Lots of `.md` which uses `result` in specs would change.